---
layout: page
title: CV
---

## Table of Contents
1. [Education](#education)
2. [Honors](#honors)
3. [Professional Experience](#professional)
4. [Research Experience](#research)
5. [Teaching Experience](#teaching)
6. [Grants](#grants)
7. [Languages and Tools](#languages)
8. [Publications](#publications)
9. [Presentations](#presentations)
10. [Professional Memberships](#memberships)

<a id="education"></a>
###EDUCATION:  
B.S.	University of Arizona, Honors College  (May 2013)  
*Majors:* Computer Science; Molecular and Cellular Biology
*Minor:* Mathematics  
*Cumulative GPA:* 3.926  
*Study abroad*	National University of Singapore, Fall 2011  
*NSF REU*	University of Pittsburgh Department of Computational and Systems Biology, Summer 2011  

<a id="honors"></a>
###HONORS:   
**University of Arizona:**  
Pillars of Excellence Awardee, Spring 2013  
College of Science Outstanding Senior, Fall 2012  
Computer Science Department Outstanding Senior, Fall 2012  
Women in Science and Engineering Student Excellence Award, Spring 2012  
Galileo Circle Scholarship in Computer Science, Spring 2011  
Lund-Wolfe Computer Science Scholarship, 2010-11 academic year  
**National Honors:**  
Phi Beta Kappa, Spring 2012  	
Phi Kappa Phi, Spring 2012  
National Merit Scholar, Spring 2008  

<a id="professional"></a>
###PROFESSIONAL EXPERIENCE:  
*Computer Science Education Coordinator, California State University Monterey Bay, July 2013-Present* 
 
 * Support students in computer science skill development and technical career preparation, with special focus on students who are part of a cohort-based program.  
 * Train 30 computer science tutors weekly in active learning techniques in computer science, and develop materials for use in tutor-guided small-group learning.  
 * Critically analyze computer science curriculum and develop supplemental activities to ensure students receive an education on par with higher-ranked institutions. 

<a id="research"></a>  
###RESEARCH EXPERIENCE:   
*Faster Optimization for ∂a∂i, University of Arizona Molecular and Cellular Biology, Summer 2012-Spring 2013*  

- Developed strategies to improve nonlinear optimization routines for the open source demographic inference software package ∂a∂i, with a focus on tweaking SciPy’s BFGS algorithm for ∂a∂i’s objective function.  
- Benchmarked the performance of optimizers for typical ∂a∂i models and data, and fit demographic inference models for African populations.  

*New Gene Microarray Clustering Techniques, University of Pittsburgh TECBio REU, Summer 2011*  

- Utilized k-means clustering and Markov blankets to predict connections between gene microarray data and visual traits of liver tumors.  
- Implemented the Partition Decoupling Method in MATLAB, and utilized it to compare the effects of microRNA regulatory elements and transcription factor regulatory elements.  

*Epigenetics of Asthma Analysis, University of Arizona Computer Science, Spring 2011*
  
- Developed a C program that locates genes, regulatory elements, and CpG islands that overlap genetic regions of interest.  
- Consulted in the development of an integer linear programming algorithm examining the relation of methylation levels across the genome to asthma prevalence.  

*Lombardi Graph Drawing, University of Arizona Computer Science, Spring-Fall 2011*  

 * Developed a force-directed algorithm for visualizing graphs with perfect angular resolution and circular arcs. Implemented our algorithm and another proposed algorithm in Python. (lombardi.cs.arizona.edu)  

*Utility of "Flats" Calibration in Extrasolar Planet Detection, University of Arizona Astronomy Club, Fall 2011*  

 * Performed statistical analysis of the effects of telescope calibration on extrasolar planet detection, using R. Determined that “flat” calibrations had no significant effect on signal-to-noise ratio, saving the team hours in telescope calibration time.

*LoCuS Labs for Computer Science, University of Arizona Computer Science, Summer 2010*  

 * Conceptualized and developed extendable virtual notebook system that can be customized with quizzes, images, and interactive activities, with feedback from K12 teachers. Programming was done in Java. (cs.arizona.edu/projects/focal/ergalics/fieldguide/)

<a id="teaching"></a>
###TEACHING EXPERIENCE:  
*Instructor, Cohort Enrichment - Year 1, CSin3 program, Hartnell College, Fall 2013-Present*  

 * Lead three-hour weekly session for freshmen that covers programming fundamentals, study skills, teambuilding, culture building, career exploration, and resume creation.  
 * Guide Peer Instruction and pair programming activities at the introductory programming level.

*Instructor, Data Structures Boot Camp, CSin3 program, Hartnell College, Summer 2015*  
  
 * Taught intensive 2-week boot camp on data structures basics, including binary search, Big-O, linked lists, stacks, queues, and recursion.  
 * Utilized active lecture style featuring Peer Instruction. All assignments incorporated unit testing.

*Teaching Assistant, 300-level quantitative biology course, University of Arizona, Spring and Fall 2012*  

 * Held weekly office hours, often with an emphasis on MATLAB help.  
 * Developed answer sets and graded homework assignments.  
 * Created MATLAB tutorials accessible to students without programming experience.  

*Director, Women in Computer Science’s programming classes for non-majors, Spring 2012*  

 * Supervised 11 WiCS members to advertise, plan, and teach basic programming in Python, Java, and C over multiple weekend workshops. 

<a id="grants"></a>
###GRANTS:  
*Innovation Award, California Department of Finance, $5 millon for continued work and expansion of the CSin3 program*

<a id="languages"></a>
###PROGRAMMING LANGUAGES:   
Python with SciPy/NumPy/Matplotlib, MATLAB, Java, C, C++, MIPS
###DEVELOPMENT TOOLS:  
git, bash, vagrant

<a id="publications"></a> 
###PUBLICATIONS:  
1.	K. Cunningham, A. Seetharam, S. Narayanan, B. Su. “Establishing an Effective Teaching Assistant Program for Computer Science Undergraduate Education.” 14th Annual Hawaii International Conference on Education, 2016. (In press)
2.	G.T. Huang, K. Cunningham, P.V. Benos, C. Chennubhotla. “Spectral clustering strategies for heterogeneous disease expression data.” Pacific Symposium on Biocomputing, 2013.
3.	J. Turner, B. Smart, K. Hardegree-Ullman, T. Carleton, A. Walker-LaFollette, B. Crawford, C. Smith, A. McGraw, L. Small, M. Rocchetto, K. Cunningham, et al. “Near-UV and optical observations of the transiting exoplanet TrES-3b.” Monthly Notices of the Royal Astronomical Society. 2012 September.
4.	R. Chernobelskiy, K. Cunningham, M. Goodrich, S. G. Kobourov, L. Trott, “Force-Directed Lombardi-Style Graph Drawing.” 19th Symposium on Graph Drawing (GD), p. 320-331, 2011.

###PAPERS IN PROGRESS:  
1.	M. Lara, K. Cunningham, B. Su. “Implementing Peer-Led Team Learning in a Community College Computer Science Course: Lessons Learned.” (In progress)
2.	S. Narayanan, B. Su, K. Cunningham. “CSin3: A Three-Year, Cohort-Based Computer Science Degree Program.” (In progress)

<a id="presentations"></a> 
###PRESENTATIONS:  
  1. K. Cunningham. “Peer-Led Team Learning Supports Computer Science Persistence at Community College.” Poster session presented at: International Computing Education Research Conference, 2015.
  2. K. Cunningham, D. Fossati. “Exploring Discovery and Refinement of KCs with Introductory Programming Assignment Data.” Poster summarized research project completed at: LearnLab Summer School, 2015 at Carnegie Mellon University.
  3. R. Chernobelskiy, K. Cunningham, M. Goodrich, S. G. Kobourov, L. Trott, “Force-Directed Lombardi-Style Graph Drawing.” Conference publication presented at 19th Symposium on Graph Drawing (GD), p. 320-331, 2011.

<a id="memberships"></a> 
###PROFESSIONAL MEMBERSHIPS:  
 * ACM – Member since July 2013
 * SIGCSE – Member since February 2015
 * CSTA – Member since July 2013

